cpp/change-notes/2020-02-04-unsigned-difference-expression-compared-zero.md

```diff
@@ -1,2 +1,2 @@
 lgtm
-* A new query (`cpp/unsigned-difference-expression-compared-zero`) has been added. The query finds unsigned subtractions used in relational comparisons with the value 0.
\ No newline at end of file
+* A new query (`cpp/unsigned-difference-expression-compared-zero`) has been added. The query finds unsigned subtractions used in relational comparisons with the value 0. This query was originally submitted as an experimental query by @ihsinme in https://github.com/github/codeql/pull/4745.
```

Comment:
```suggestion
* A new query (`cpp/unsigned-difference-expression-compared-zero`) is run but not yet displayed on LGTM. The query finds unsigned subtractions used in relational comparisons with the value 0. This query was originally submitted as an experimental query by @ihsinme in https://github.com/github/codeql/pull/4745.
```

We record in changes notes when LGTM's set of run/displayed queries change. Example: https://github.com/github/codeql/blob/main/change-notes/1.23/analysis-cpp.md